labs/lab02/skel/win/2-crc/crc.c

```diff
@@ -20,6 +20,11 @@
 #define BUFSIZE 512
 #define CHUNKSIZE 32
 
+static void WriteCrc(int crc, HANDLE hWrite)
+{
```

Comment:
Should me add some vars here?

ex (or something similar): 
```
BOOL bRet;
DWORD dwBytesWritten, dwBytesToWrite, dwBytesTotalWritten;
```

Reply:
I also added similar variables to the `ReadChunk` function.